release/check-image-vulnerabilities/check-image-vulnerabilities.sh

```diff
@@ -0,0 +1,129 @@
+#!/usr/bin/env bash
+#
+# Counts the number of fixable vulnerabilities in the scan result.
+#
+# Local run:
+#
+#  test/local-env.sh release/check-image-vulnerabilities/check-image-vulnerabilities.sh <image> <version>
+#
+set -euo pipefail
+
+IMAGE="${1:-}"
+VERSION="${2:-}"
+SUMMARY_PREFIX="${3:-}"
+
+check_not_empty \
+    IMAGE \
+    VERSION \
+    SUMMARY_PREFIX
+
+function main() {
+  local image="$1"
+  local version="$2"
+  local summary_prefix="$3"
```

Comment:
It feels so redundant here.
If you don't actually need the global variables, just call `main` as
```bash
main "$@"
```
and do argument assignment with validation inside.